version.props

```diff
@@ -8,7 +8,7 @@
     <FeatureBranchVersionPrefix Condition="'$(FeatureBranchVersionPrefix)' == ''">a-</FeatureBranchVersionPrefix>
     <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(FeatureBranchVersionSuffix)' != ''">$(FeatureBranchVersionPrefix)$(VersionSuffix)-$([System.Text.RegularExpressions.Regex]::Replace('$(FeatureBranchVersionSuffix)', '[^\w-]', '-'))</VersionSuffix>
     <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
-    <AspNetCoreModuleVersionMajor>8</AspNetCoreModuleVersionMajor>
+    <AspNetCoreModuleVersionMajor>12</AspNetCoreModuleVersionMajor>
     <AspNetCoreModuleVersionMinor>1</AspNetCoreModuleVersionMinor>
     <AspNetCoreModuleVersionRevision>0</AspNetCoreModuleVersionRevision>
```

Comment:
Will this or the minor version need to change in future servicing updates? If so, I'd suggest tying these to the value of `VersionPrefix`.

Reply:
Nvrm, versions are on a repo by repo basis.